src/components/Main/Main.js

```diff
@@ -0,0 +1,94 @@
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import chunk from 'lodash/chunk';
+import nth from 'lodash/nth';
+import MovieListItem from '../common/MovieListItem';
+import MovieCard from '../common/MovieCard';
+import { getMovies } from './selectors';
+import { toggleFavorite, deleteMovie } from '../../actions/MoviesActions';
+import { playMovie } from '../../actions/PlayerActions';
```

Comment:
1. nie masz ścieżek w stylu:
../../../../../actions/WhateverActions.js
2. łatwiej zarządzać ścieżkami, gdy musisz wprowadzić jakąś zmianę wynikającą ze zmiany położenia pliku/modułu, łatwiej tę zmianę wprowadzić wszędzie, gdzie plik jest importowany

Nie, tylko jeżeli jest to odwołanie większe niż ../